channels/pe.m3u

```diff
@@ -184,7 +184,7 @@ https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtu
 #EXTINF:-1 tvg-id="RadioXtrema.pe" tvg-country="PE" tvg-language="Spanish" tvg-logo="" group-title="Music",Radio Xtrema [Not 24/7]
 https://rtmp02.portalexpress.es/xtrematv2/xtrematv2/playlist.m3u8
 #EXTINF:-1 tvg-id="RadioTVOrienteYurimaguas.pe" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/www.roriente.org/picture?width=320&height=320" group-title="Local",RadioTV Oriente (Yurimaguas) (360p) [Not 24/7]
```

Comment:
```suggestion
#EXTINF:-1 tvg-id="RadioTVOrienteYurimaguas.pe" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/www.roriente.org/picture?width=320&height=320" group-title="Local",RadioTV Oriente (Yurimaguas) (1080p)
```

Reply:
I forgot it. Thank you for this suggestion.